### ISSUE TITLE
Demo broken

### ISSUE THREAD

**Issue by dfabulich** [PolymerLabs/lit-html#156]
Here's what I see when I open clock.html in Chrome 62:
```
lit-html.ts:209 Uncaught (in promise) TypeError: Cannot read property 'value' of null
    at new Template (lit-html.ts:209)
    at litTag (lit-html.ts:55)
    at html (lit-html.ts:36)
    at HTMLElement.render (clock.js:35)
    at Promise.resolve.then (clock.js:119)
    at <anonymous>
```

**Comment by caseycarroll42** [comment]
I am seeing this as well!

**Comment by justinfagnani** [comment]
The demo's working for me on master, we may just need a release. What version of lit are you using?

**Comment by dfabulich** [comment]
Looks like you fixed it in revision 072ad42efd3ad12a37c02423280afabb301f8dfe
